src/driver/gen_rpc_driver_ssl.erl

```diff
@@ -16,11 +16,11 @@
 %%% Include the HUT library
 -include_lib("hut/include/hut.hrl").
 %%% Include this library's name macro
--include("app.hrl").
+-include("../app.hrl").
```

Comment:
Compiles AND works with ex_doc (there's an issue identified for this, in rebar3_ex_doc/GitHub, but for the time being I believe this is a good compromise).